charts/incubator/vocechat-server/values.yaml

```
@@ -0,0 +1,32 @@
+image:
+  repository: tccr.io/truecharts/vocechat-server
+  pullPolicy: IfNotPresent
+  tag: v0.3.38@sha256:a2af75f54ed922a0803f14d0839a25d51d5bc0f6127e972bcf5ee798efcfa5b8
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        port: 3009
+
+vocechat:
+  frontend_url: localhost:3009
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          args:
+            - --network.bind 0.0.0.0:{{ .Values.service.main.ports.main.port }}
+            - --network.frontend_url "{{ .Values.vocechat.frontend_url }}"
```

Are those getting picked up?

args usually are one line the flag and one line the value
```suggestion
            - --network.bind 
            - 0.0.0.0:{{ .Values.service.main.ports.main.port }}
            - --network.frontend_url 
            - {{ .Values.vocechat.frontend_url }}
```

---

[yes](https://github.com/truecharts/charts/actions/runs/6283050727/job/17063179080?pr=12866#step:10:223-226) but ill commit this though since its cleaner